core/modules/user.py

```diff
@@ -47,6 +57,16 @@ class UserSkel(skeleton.Skeleton):
         unique=UniqueValue(UniqueLockMethod.SameValue, False, "UID already in use"),
     )
 
+    synced = BooleanBone(
```

Comment:
```suggestion
    sync = BooleanBone(
```
Please rename this bone to `sync`.